### PR TITLE
10.0 feature/2 smtp per user compat 10 evi

### PR DESCRIPTION
**Pull request by boolit-eivydas**

### CODE REVIEW

File: smtp_per_user/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# See LICENSE file for full copyright and licensing details.
+
+{
+    'name': "SMTP Per User",
+    'version': '1.0.0',
+    'summary': 'Send letters from Odoo using your own mail',
+    'category': 'Mail',
+    'description': """Can configure different mail servers per user""",
+    'author': 'OERP',
+    'license': 'AGPL-3',
```
Comment (oerp-odoo) [inline smtp_per_user/__manifest__.py]: On Odoo 10, we should use LGPL-3 license.

File: smtp_per_user/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    'website': "www.oerp.eu",
```
Comment (oerp-odoo) [inline smtp_per_user/__manifest__.py]: website provided is out of date

File: smtp_per_user/static/description/index.html [hunk 3]
```diff
@@ -6,7 +6,7 @@ <h3 class="oe_slogan">Send letters from Odoo using your own mail</h3>
...
+Can configure different mail servers per user. Can specify email name for every server. For example - Email Name: <i>'My Custom Name'</i> and sending mail is <i>example@example.com</i>. When sent, it will show like this: <i>"My Custom Name example@example.com"</i>. Specific mail server can be set to ignore owner mail server and use forced server which was chosen by priority or by other means. This is forked and improved version of <a href="https://github.com/bamps/smtp-per-user">smpt-per-user</a> for Odoo 10.0.
             </p>
             <div class="oe_centeralign oe_websiteonly">
                 <a href="http://www.oerp.eu/#trial" class="oe_button oe_big oe_tacky">Start your <span class="oe_emph">free</span> trial</a>
```
Comment (oerp-odoo) [inline smtp_per_user/static/description/index.html]: This link is out of date

**Review by oerp-odoo — COMMENTED** [review]
Why it is pointing to `10.0-feature/2-smtp-per-user-compat-10` branch? Shouldn't you point to develop branch?

File: smtp_per_user/static/description/index.html (hunk 3 above)
Comment (boolit-eivydas) [reply smtp_per_user/static/description/index.html]: Agree. Updated.

File: smtp_per_user/models/smtp_per_user.py [hunk 4]
```diff
@@ -55,4 +39,4 @@ def send(self, auto_commit=False, raise_exception=False):
                     if mail_server:
                         email.mail_server_id = mail_server.id
             email.email_from = email.mail_server_id.replace_email_name(email.email_from)
-        return super(mail_mail, self).send(auto_commit=False, raise_exception=False)
+        return super(MailMail, self).send(auto_commit=False, raise_exception=False)
```
Comment (oerp-odoo) [inline smtp_per_user/models/smtp_per_user.py]: This can't be right. It should pass arguments, not specify exact values.

**Comment by boolit-eivydas** [comment]
This is not pointing to master, because if done so, the full revert commit would be visible and not just the changes that make this module compatible.

**Review by oerp-odoo — APPROVED** [review]

**Pull request merged**